src/firebase.ts

```diff
@@ -64,3 +75,13 @@ export function useEmulatedFirebaseApp(
 
   return app;
 }
+
+const loadCloudWorkstationCookies = async (url?: string) => {
+  if (url && url.includes('cloudworkstations.dev')) {
```

Comment:
Can we just re-use the `isCloudWorkstation` call here?

Reply:
➕